includes/callbacks.inc

```diff
@@ -90,20 +91,21 @@ function islandora_entities_autocomplete($type) {
     'fl' => array($mappings[$type]['title'], 'PID'),
```

Comment:
Probably doesn't matter, as the entire Solr record is included under `solr_doc` I can remove these. But as they don't affect functionality, I'd rather leave them. I think a subsequent PR to clean up bad function signatures, do a full module code sniff, etc is in order.